deploy/future

```diff
@@ -0,0 +1,172 @@
+#!/bin/bash
+# Don't run this with -e, grep exits with 1 when it finds no results
+
+function check_py () {
+# This is a bit janky. If you know how to get the output
+# to go on separate lines, please tell me.
+    results=`git --no-pager grep $1|grep "\.py"|grep -v "future-ok"`
```

Comment:
I wouldn't worry about the collision?  But if you want to keep it, I think `# nofuture` is what I would expect, if it is a bit amusing.